src/markdown-pages/docs/cli-command-reference/cli-command-reference.md

```diff
@@ -6,37 +6,18 @@ title: "Getting started using the Galasa CLI"
 
 If you are installing Galasa for using in the command-line, you can use the Galasa command line interface tool (Galasa CLI) to interact with Galasa to complete tasks, for example, submitting and monitoring Galasa test runs. You can use the same set of Galasa CLI commands to run a given task, regardless of the technology that you are using. 
 
+Read on to understand more about the prerequisites that you need to install so that you can start using the Galasa CLI.
 
-## About Galasa CLI commands
 
-Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+## Prerequisites
 
-You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+| Software |  Description  |
+| :---- | :-------- | 
+| Java JDK  | Required. Galasa tests and Managers are written in Java - you need to install a Java version 11 JDK or later to use it. _Note:_ We do not currently support Java 17 or later. |
+| Maven or Gradle  | You must install either Maven or Gradle in order to build Galasa projects. Galasa projects are hierarchical file structures that provide the ability to store and run Galasa tests. All Galasa versions are compatible with Gradle releases 6.8.x and later. |
```

Comment:
Not compatable with later versions of gradle. Thought we were fixing that.

Reply:
yes, that was fixed. think a problem with merge. now resolved.